History.md

```diff
@@ -6,6 +6,7 @@
 
 * Bugfixes
   * Fix Errno::EINVAL when SSL is enabled and browser rejects cert (#1564)
+  * Use assert_operator for better error messages (#2019)
```

Comment:
I don't think improvements of the tests need (or should) be mentioned as a bugfix.

You can add the text `[changelog skip]` to the pull request title to skip the changelog check.

Reply:
Maybe I can open a different PR to include a hint on CI :D